platform/android/export/gradle_export_util.h

```diff
@@ -44,28 +44,6 @@ const String godot_project_name_xml_string = R"(<?xml version="1.0" encoding="ut
 </resources>
 )";
 
-OS::ScreenOrientation _get_screen_orientation() {
```

Comment:
As shown by @akien-mga's error stacktrace, there's another use of `_get_screen_orientation()` in this file that needs to be updated.